src/client/components/buttons.tsx

```diff
@@ -13,7 +13,9 @@ interface ButtonProps {
 }
 
 export const Button = styled.button<ButtonProps>`
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
+  gap: 0.5rem;
```

Comment:
Have you checked if flex + gap has sufficient browser support?

Reply:
Yes. I mean, it seems to be well supported now. It's being used in other places in the codebase as well :)

Reply:
Hmm are you sure 😉 or are we using grid + gap because I have know that has better browser support

88% users covered it seems: https://caniuse.com/flexbox-gap
But still a Working Draft...

Reply:
🤔 Yeah, you're right. Gonna refactor it.

Reply:
I can't wait for that to be officially part of CSS though -- we should start using it as soon as it lands 😅

Reply:
Me too!